domains/mikko.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "Mikko's Developer Page",
+  "repo": "https://github.com/mikkortx/cv",
+  "owner": {
+    "username": "mikkortx",
+    "email": "mikkoahh@gmail.com",
+    "twitter": "mikkortx"
+  },
+  "record": {
+    "CNAME": "https://mikkortx.github.io/cv/"
```

Comment:
cname cant have slash

Reply:
want a url redirection? use an url record
want a masked website? change cname value to `mikkortx.github.io`.

Reply:
sure please

Reply:
have you resolve it